transform/transformer.go

```diff
@@ -233,12 +235,24 @@ func (t *LogTransformer) transformDiff() {
 	t.LogContent = strings.Join(transformedLines, "\n")
 }
 
-// truncate to avoid Message:Body is too long (maximum is 65536 characters)
+// truncate to avoid Message:Body is too long (maximum is set per VCS)
 func (t *LogTransformer) truncate() {
+	var maxCommentLength int
+	if t.Vcs == config.VcsGithubEnterprise && t.GithubMaxCommentLength != 0 {
```

Comment:
could test cases to cover these branches? https://github.com/karlderkaefer/cdk-notifier/blob/6bf0b493884526bf0215432d50d79d9d25580016/transform/transformer_test.go#L199

Reply:
Done